### PR TITLE
Merge xcode10.2 into master

### PR DESCRIPTION
**Pull request by VincentDondain**

### CODE REVIEW

File: Make.config [hunk 1]
```diff
@@ -72,16 +72,16 @@ MAX_SHARPIE_VERSION=3.4.99
...
 
 # Minimum OSX versions for building XI/XM
-MIN_OSX_BUILD_VERSION=10.13
+MIN_OSX_BUILD_VERSION=10.14
 # Minimum OSX version for executing XI/XM tooling.
 MIN_OSX_VERSION_FOR_IOS=10.11
 MIN_OSX_VERSION_FOR_MAC=10.11
 
-IOS_SDK_VERSION=12.1
+IOS_SDK_VERSION=12.2
```
Comment (chamons) [inline Make.config]: Shouldn't master be odd version numbers?

**Comment by VincentDondain** [comment]
We're stuck on `System.Security.Cryptography.Algorithms.dll` for some reason. cc @mandel-macaque

**Comment by VincentDondain** [comment]
build

**Comment by rolfbjarne** [comment]
Test failure are unrelated:

* */watchOS: https://github.com/xamarin/maccore/issues/581

The breaking API changes are a known issue we currently can't fix (enum value changed).

**Pull request merged**